### PR TITLE
Suggest similar pools based on weights, fee, and liquidity

### PR DESCRIPTION
**Pull request by Destiner**

### CODE REVIEW

File: src/components/Message/SimilarPools.vue [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+<script>
+import { mapActions } from 'vuex';
+
+const WEIGHT_FACTOR = 0.0025;
+const FEE_FACTOR = 40;
+const LIQUIDITY_THRESHOLD = 100000;
+const LIQUIDITY_FACTOR = 0.1;
+const SCORE_THRESHOLD = 0.1;
+
+function getPoolScore(pool, modelPool) {
```
Comment (bonustrack) [inline src/components/Message/SimilarPools.vue:30]: I think it's better to keep the views and component logic as light as possible, the function `getPoolScore` and `processPools` can be on `utils.ts` or on a separated file, i believe we will want to reuse these functions on others part of the site, the factor could be visible on Pool > About for example.

File: src/components/Message/SimilarPools.vue [hunk 2]
```diff
@@ -0,0 +1,162 @@
+<template>
+  <div v-if="pools.length > 0" id="info" class="d-flex flex-items-center p-4">
+    <Icon name="warning" size="22" class="mr-4" />
+    <div>
+      <div>
+        There are similar pools available:
+      </div>
+      <div v-for="pool in pools" :key="pool.id">
+        • {{ formatPool(pool) }}
```
Comment (bonustrack) [inline src/components/Message/SimilarPools.vue:9]: Can you start the link here so the user can't miss it. Also add a target="_blank" so the user dont his creation pool form when he visit a similar pool.

Comment (Destiner) [reply src/components/Message/SimilarPools.vue]: Changed.

File: src/components/Message/SimilarPools.vue (hunk 1 above)
Comment (Destiner) [reply src/components/Message/SimilarPools.vue:31]: Hmm agree in general, but not sure whether it applies here. This code is very specific to pool recommendation. Besides, I think `utils.ts` is starting to become bloated. However, happy to move some of this code to `helpers` once we need it elsewhere.

**Pull request merged**